### ISSUE TITLE
Absolute paths in stats.json

### ISSUE THREAD

**Issue by t0x1c123** [sysgears/spinjs#35]
Hello, I have a little problem. I'm using spinjs with your apollo devstack. I encoutered problem with build script. I'm using teamcity for deployment. But when I build for example on server with build agent and then copy build directory to other server I'm getting error beacuse of absolute paths to node modules in builded files. So I decided to use other way and that is call yarn build command via SSH on destination server. But yarn build is doing two things and that is build backend and build frontend. And there is another problem beacause as soon as backend build finishes teamcity get exit error and thinks that build step is finished. So maybe there would be better to have something like yarn build backend and yarn build frontend commands separatly?

This is the error whem copying build directory from build server to other server.

`$ node --harmony build/server
Error: Cannot find module '!../../../node_modules/isomorphic-style-loader/lib/in
sertCss.js'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at require (internal/module.js:20:19)`

**Comment by vlasenko** [comment]
@t0x1c123 When you copy `build` directory to other server - the backend part still does not include code from `node_modules`, it expects that you have at least production `node_modules` installed. To do so on the server to which you have copied `build` dir, run:
`yarn install --prod`

**Comment by t0x1c123** [comment]
@vlasenko Yes I know. I do that. I have complete copy of project including node_modules. One and only differnce is that i dont build there. So if i copy build directory from other server into folder which contains the same node_modules it fails with error above.

**Comment by vlasenko** [comment]
@t0x1c123 Did you provide full stack trace in this issue report?

**Comment by vlasenko** [comment]
@t0x1c123 Please provide full stack trace in the issue report and use tripple backslashes to format your code here

**Comment by t0x1c123** [comment]
Here you go
```
C:\inetpub\wwwroot.bit.contractwatcher\newFrontend>yarn start
yarn run v1.3.2
$ node --harmony build/server
Error: Cannot find module '!../../../node_modules/isomorphic-style-loader/lib/in
sertCss.js'
    at Function.Module._resolveFilename (module.js:469:15)
    at Function.Module._load (module.js:417:25)
    at require (internal/module.js:20:19)
    at Object.<anonymous> (C:\inetpub\wwwroot.bit.contractwatcher\newFrontend\bu
ild\server\index.js:2:1861165)
    at t (C:\inetpub\wwwroot.bit.contractwatcher\newFrontend\build\server\index.
js:2:101)
    at Object.<anonymous> (C:\inetpub\wwwroot.bit.contractwatcher\newFrontend\bu
ild\server\index.js:2:283588)
    at t (C:\inetpub\wwwroot.bit.contractwatcher\newFrontend\build\server\index.
js:2:101)
    at Object.<anonymous> (C:\inetpub\wwwroot.bit.contractwatcher\newFrontend\bu
ild\server\index.js:2:210928)
    at t (C:\inetpub\wwwroot.bit.contractwatcher\newFrontend\build\server\index.
js:2:101)
    at Object.<anonymous> (C:\inetpub\wwwroot.bit.contractwatcher\newFrontend\bu
ild\server\index.js:2:4286)
    at t (C:\inetpub\wwwroot.bit.contractwatcher\newFrontend\build\server\index.
js:2:101)
    at C:\inetpub\wwwroot.bit.contractwatcher\newFrontend\build\server\index.js:
2:452
    at Object.<anonymous> (C:\inetpub\wwwroot.bit.contractwatcher\newFrontend\bu
ild\server\index.js:2:462)
    at Module._compile (module.js:570:32)
    at Object.Module._extensions..js (module.js:579:10)
    at Module.load (module.js:487:32)
    at tryModuleLoad (module.js:446:12)
    at Function.Module._load (module.js:438:3)
    at Module.runMain (module.js:604:10)
    at run (bootstrap_node.js:389:7)
    at startup (bootstrap_node.js:149:9)
    at bootstrap_node.js:504:3
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this comm
and.
```

**Comment by vlasenko** [comment]
@t0x1c123 I think there is some problem with your code. It seems to be doing this require: `!../../../node_modules/isomorphic-style-loader/lib/insertCss.js`

**Comment by vlasenko** [comment]
@t0x1c123 And this is a dev dependency. So in case you do this, you should install not only production node_modules, but also dev node_modules, or move `isomorphic-style-loader` into production dependencies. In any case this has nothing to do neither with `spinjs` nor with `apollo-universal-starter-kit`. It is only your code

**Comment by vlasenko** [comment]
@t0x1c123 Actually I think I was wrong. Backend produced by `yarn build` of `apollo-universal-starter-kit` relies on `devDependencies` too. So you should really install both devDependencies and dependencies in order for it to work. E.g. run just `YARN_PRODUCTION=false yarn install` to install all `node_modules` for sure on your server.

**Comment by vlasenko** [comment]
I think we should really not depend on presence of `node_modules` for backend files. I will try to fix that soon

**Comment by vlasenko** [comment]
@t0x1c123 It is not really easy for me to test that current kit actually relies on `devDependencies`. Maybe it is only your code, but I cannot be sure. If you install both `devDependencies` and `dependencies` on your server it should work in any case. Also check whether you require `!...isomorphic-style-loader` directly from your code and try to not doing that

**Comment by t0x1c123** [comment]
@vlasenko Yeah it's weird but I dont import that package anywhere in my code. I also have installed all dependencies. Node modules folder contains that plugin and also that file so it's very weird.. Also my webpack ovveride file is in default state. If I search for that package only file that contains that name is in build folder

**Comment by vlasenko** [comment]
@t0x1c123 Ok, I will look at this issue first thing tomorrow morning

**Comment by vlasenko** [comment]
@t0x1c123 I have checked the latest version of the kit and it runs fine for me when files from `build` moved to other folder and you copy production sqlite3 db to that folder and run `yarn --prod`. The only nuisance was that `babel-polyfill` is not install, so I had to move it from devDependencies to dependencies and run `yarn --prod` again. After that the server launched succesfuly and worked as expected. So the issue your are describing here is not reproduced for me. Hence I think it is something specific to your modifications to the kit. I will need detailed reproduction steps to investigate your issue further, if you are going to submit them and will close this issue for now, because it is neither a problem of the `spinjs` nor of the `kit`
